vllm/model_executor/models/llama.py

```diff
@@ -644,14 +645,17 @@ class LlamaEmbeddingModel(nn.Module, SupportsPP):
 
     def __init__(
         self,
-        pooler_config: Optional[PoolerConfig] = None,
+        vllm_config: VllmConfig,
+        prefix: str = "",
         **kwargs,
     ) -> None:
         super().__init__()
 
-        self.model = LlamaModel(**kwargs)
+        self.model = LlamaModel(vllm_config.model_config.hf_config,
```

Comment:
Should also pass the other sub-configs to the inner model..

Reply:
yes that will be the next step. for this pr, we first pass the config to the top-level models.